requirements/pip.txt

```diff
@@ -99,3 +99,6 @@ packaging==17.1
 # Commenting stuff
 django-cors-middleware==1.3.1
 nilsimsa==0.3.8
+
+# Git dependencies
+git+https://github.com/italia/docs-italia-convertitore-web.git@master#egg=docs-italia-convertitore-web
```

Comment:
Userei `docsitalia.txt` https://github.com/italia/docs.italia.it/blob/italia-2.3.13/requirements/docsitalia.txt per dichiarare questa dipendenza

Reply:
Spostandolo in `docsitalia.txt` saltano i test (il requirement file non viene usato nei test ma il converter compare in `INSTALLED_APPS` e non viene trovato).
`pip.txt` è invece una base comune a produzione e testing. Volendo si può fare un refactoring delle dipendenze (il mio approccio voleva essere il meno invasivo possibile).

Reply:
ok,  ero solo preoccupato dell'eventuale upgrade, ma lo vediamo poi